Examples/Extensions/TodayWidget/TodayWidget.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.mapbox.MMETestHost</string>
```

Comment:
This is an App Group entitlement which is now shared between Today Widget and TestHost as a mechanism for sharing data between the two targets.

This will only function as expected if the app group id is also leveraged in the Defaults(suite:)